test/Round-test.js

```diff
@@ -15,17 +15,19 @@ describe('Round', () => {
       { id: 2, name: 'Peerat' },
       { id: 3, name: 'Victor' }
     ]);
-    puzzle = new Puzzle(data.puzzles);
-    puzzle.choosePuzzleBank();
-    puzzle = puzzle.choosePuzzle();
+    // puzzle = new Puzzle(data.puzzles);
+    // puzzle.choosePuzzleBank();
+    // puzzle = puzzle.setPuzzle();
```

Comment:
You will be in grave trouble on the job if you commit commented out code to a PR. This clutters up the diff and makes things a bit more difficult to review. Try to avoid this at all costs. One way to be mindful of not committing such things is to do a `git add --patch` rather than a `git add .` --- adding in patches allows you to review your changes and select exactly which ones you want to commit.